example.config.yml

```diff
@@ -271,3 +271,4 @@ selenium_version: 2.46.0
 # Other configuration.
 dashboard_install_dir: /var/www/dashboard
 known_hosts_path: ~/.ssh/known_hosts
+ssh_home: "{{ drupal_core_path }}"
```

Comment:
@danbohea - haha - I struggled with the naming of this too! + it's not a convention.

Couldn't find a standard name so since by default SSH connections drop you in the users home dir, this config for that SSH home dir.